### PR TITLE
Hide duplicate tooltip in `pr-approvals-count`

### PR DESCRIPTION
**Pull request by BrewedCoffee**
Resolves https://github.com/sindresorhus/refined-github/issues/4371

With this CSS change, the redundant tooltip is no longer present.

Tooltip for requested changes doesn't seem to be redundant as the count isn't shown inline, so this only targets 'approved' tooltips.

## Test URLs
URL: https://github.com/sindresorhus/refined-github/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc

### CODE REVIEW

File: source/features/pr-approvals-count.css [hunk 1]
```diff
@@ -17,3 +17,7 @@
 .js-issue-row .color-text-secondary [aria-label*='changes'] {
 	color: var(--github-red) !important;
 }
+
+[aria-label*='approval']:after {
```
Comment (fregante) [inline source/features/pr-approvals-count.css:21]: Let's match the existing selector and add a comment

```suggestion
/* Disable now-duplicate tooltip */
.js-issue-row .color-text-secondary [aria-label*='approval']:after {
```

**Review by fregante — APPROVED** [review]

**Comment by fregante** [comment]
Thank you for your PR Jeff!

**Comment by BrewedCoffee** [comment]
How can I run lint checks locally?

**Comment by cheap-glitch** [comment]
> How can I run lint checks locally?

`npm run lint:js` and `npm run lint:css` (or `npm run lint` for both at the same time). All scripts are in package.json

**Comment by BrewedCoffee** [comment]
Sorry to re-request, just need a maintainer to approve workflow tests.

**Comment by fregante** [comment]
Thank you for the PR Jeff! :)

**Pull request merged**